client/pool_test.go

```diff
@@ -79,12 +80,16 @@ func TestListPools(t *testing.T) {
 			Extras: model.StoragePoolExtraSpec{
 				DataStorage: model.DataStorageLoS{
 					ProvisioningPolicy: "Thin",
-					IsSpaceEfficient:   true,
+					Compression:   true,
+                                        Deduplication: true,
 				},
```

Comment:
please format this file

Reply:
I have created two issues #1080 and #1081 for tracking purpose. I have tested for nfs and lvm drivers. and will continue to test for others drivers.